src/tours/models.py

```diff
@@ -22,5 +23,14 @@ class Tours(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    def avg_rating(self):
+        sum = 0
+        ratings = Rating.objects.filter(tour_id=self)
```

Comment:
can u use sum/avg in sql in here?

Reply:
No, I can't
Because ratings is list so i use for loop to plus sum
And i want to check len(rantings)

Reply:
![image](https://user-images.githubusercontent.com/38286740/89515146-ee6de300-d800-11ea-904d-d2240a222126.png)
Sum is the same

Reply:
I understand..I just fix it..thank Phong and a.Anh